packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -556,20 +564,23 @@ export function getValidatorApi({
         // winston logger doesn't like bigint
         enginePayloadValue: `${enginePayloadValue}`,
         builderPayloadValue: `${builderPayloadValue}`,
+        consensusBlockValue: `${consensusBlockValue}`,
```

Comment:
@naviechan here (& above) the intend is to choose engine or builder block based on value, so could you cal blockValue = enginePayloadValue + consensusPayloadValue for both builder and engine block and update the if condition on L544